src/com/lilithsthrone/game/sex/sexActions/submission/SALyssiethSpecials.java

```diff
@@ -132,8 +153,7 @@ public void applyEffects() {
 			Main.game.getPlayer().setBodySize(BodySize.TWO_AVERAGE.getMedianValue());
 			Main.game.getPlayer().setHeight(168);
 			Main.game.getPlayer().setAssType(AssType.DEMON_COMMON);
-			Main.game.getPlayer().setBreastType(BreastType.DEMON_COMMON);
-			Main.game.getPlayer().setBreastSize(CupSize.DD);
+			playerGrowDemonicBreasts(CupSize.DD, CupSize.C);
```

Comment:
Please tell me, if only the breast cup size should be set aka leave the crotch cup size as is or to be set to a certain value, like I did here. Personally I'd prefer the former. And if the latter: What `CupSize` should the crotch boobs actually get?

Reply:
I did as you suggested, and changed it so that crotch-breast size is left as it is in all instances.